### ISSUE TITLE
Regarding softmax and loss

### ISSUE THREAD

**Issue by s1998** [exelban/tensorflow-cifar-10#6]
From tensorflow official documentation for tf.nn.softmax_cross_entropy_with_logits_v2:

> WARNING: This op expects unscaled logits, since it performs a softmax on logits internally for efficiency. Do not call this op with the output of softmax, as it will produce incorrect results.

It might be better to use no activation here:
>softmax = tf.layers.dense(inputs=drop, units=_NUM_CLASSES, activation=tf.nn.softmax, name=scope.name)

**Comment by exelban** [comment]
@s1998 Hi, thanks for info. Can you make a PR with this?
